seed/challenges/01-front-end-development-certification/basic-bonfires.json

```diff
@@ -333,8 +333,9 @@
     },
     {
       "id": "afcc8d540bea9ea2669306b6",
+      "title": "Repeat a string repeat a string",
       "description": [
-        "Repeat a given string (first argument) n times (second argument). Return an empty string if n is a negative number.",
+        "Repeat a given string (first argument) <code>n</code> times (second argument). Return an empty string if <code>n</code> is a negative number.",
```

Comment:
Huh, shouldn't this say `num` times?

I know that was not the original  text, but would you mind updating it to say that?  Otherwise this looks good.